koku/api/report/ocp/provider_map.py

```diff
@@ -796,6 +796,17 @@ def __init__(self, provider, report_type, schema_name):
                             "request_memory": Sum("pod_request_memory_gigabyte_hours") / 24,
                             "request_cpu_units": Max(Value("Core", output_field=CharField())),
                             "request_memory_units": Max(Value("GiB", output_field=CharField())),
+                            "cost_total_distributed": self.cloud_infrastructure_cost
```

Comment:
Do we not need to add this to the annotations too? So we get the distribution breakdown at each vm level.